src/rtc/h5_prep.py

```diff
@@ -302,18 +312,124 @@ def get_metadata_dict(product_id: str,
-             datetime.now().strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
-             'Processing date and time'],
+              datetime.now().strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
+              'Processing date and time in the format YYYY-MM-DDTHH:MM:SSZ'],
         'identification/productVersion':
             ['product_version', product_version, 'Product version'],
         'identification/softwareVersion':
             ['software_version', str(SOFTWARE_VERSION), 'Software version'],
-        # 'identification/CEOSDocumentIdentifier':
-        #     ["https://ceos.org/ard/files/PFS/NRB/v5.5/CARD4L-PFS_NRB_v5.5.pdf",
-        #      'CEOS document identifier'],
+        #'identification/CEOSDocumentIdentifier':
+        #    ["https://ceos.org/ard/files/PFS/NRB/v5.5/CARD4L-PFS_NRB_v5.5.pdf",
+        #     'Product version'],
+        'RTC/metadata/sourceData/numberOfAcquisitions': # 1.6.4
+            ['number_of_acquisitions',
+             1,
+             'Number of source data acquisitions'],
+        # TODO Review: should we expose this parameter in the runconfig?
+        # 'RTC/metadata/sourceData/dataAccess':
+        #     ['source_data_access',
+        #      'https://search.asf.alaska.edu/',
+        #      'Data access URL'],
+        'RTC/metadata/sourceData/radarBand':  # 1.6.4
+            ['radar_band', 'C', 'Radar band'],
+        'RTC/metadata/sourceData/processingFacility': #  1.6.6
+            ['source_data_processing_facility',
+             (f'organization: \"{burst_in.burst_misc_metadata.processing_info_dict["organisation"]}\", '
+              f'site: \"{burst_in.burst_misc_metadata.processing_info_dict["site"]}\", '
+              f'country: \"{burst_in.burst_misc_metadata.processing_info_dict["country"]}\"'),
+             'Source data processing facility'],
+        'RTC/metadata/sourceData/processingDateTime':  # 1.6.6
+            ['source_data_processing_date_time',
+             burst_in.burst_misc_metadata.processing_info_dict['stop'],
+             'Processing date and time of the source data'],
+        'RTC/metadata/sourceData/softwareVersion':  # 1.6.6
+            ['source_data_software_version',
+             str(burst_in.ipf_version),
+             'IPF version of the source data'],
+        
+        'RTC/metadata/sourceData/azimuthLooks':  # 1.6.6
+            [None,
+             burst_in.burst_misc_metadata.azimuth_looks,
+             'Azimuth number of looks'],
+        'RTC/metadata/sourceData/slantRangeLooks':  # 1.6.6
+            [None,
+             burst_in.burst_misc_metadata.slant_range_looks,
+             'Slant range number of looks'],
+
+        'RTC/metadata/sourceData/productLevel':  # 1.6.6
+            ['source_data_product_level',
+             'L1',
+             'Product level of the source data'],
+        'RTC/metadata/sourceData/geometry':  # 1.6.7
+            ['source_data_geometry',
+             'slant range',
+             'Geometry of the source data'],
```

Comment:
```suggestion
        # 'RTC/metadata/sourceData/geometry':  # 1.6.7
        #     ['source_data_geometry',
        #     'slant range',
        #     'Geometry of the source data'],
```
Let's comment this out